### PR TITLE
Added new hardware selection page

### PR DESCRIPTION
**Pull request by maybenikhil**
Instead of focusing on detailing all hundred plus recievers and the tens of transmitters, condensed down to a guide of manufacturers and features to look out for. Feel free to comment if I missed anything, this is a long time in the making and I finally had the time to work on it.

### CODE REVIEW

File: docs/hardware/hardware-selection.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+??? info "PA/LNA"
+
+    A Low Noise Amplifier directly adds to your incoming RSSI. Typical gains are in the ballpark of +12dBm which will be observed in the RSSI as being 12dBm higher than it would have been without the LNA. This is because the LNA amplifies the incoming signal coming from the antenna before going to the RF chip, which increases the sensitivity of the receiver by boosting the incoming signal. An LNA also boosts the noise by the same amount so the sensitivity limit will likely be higher than the value quoted by the Lua.
+
+    A Power Amplifier boosts the outgoing signal strength and extends the telemetry range back to the TX. Without a PA, the power output is limited by the RF chip's max power output itself (around +13dBm 20mW). It works the same way as turning up the power output on the transmitter module, however it is not adjustable. The receiver's output is always run at max power, since it runs a much lower duty cycle than the transmitter does (duty cycle = telemetry ratio). Higher numbers are better. Most PAs are 20dBm/100mW, with some being up to 23dBm.
+
+    Recievers with a PA/LNA will have a listed telemetry output power in dBm or mW
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:104]: "Receivers"

File: docs/hardware/hardware-selection.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+??? info "Size"
+
+    The FPV world shook when ELRS released receivers that were half the size of "nano" sized receivers, included the antenna onboard, and still had kilometers of range at 250Hz/100mW. A small receiver can fit in tight places, but remember that tucking a tiny receiver's ceramic antenna deep inside a stack behind carbon reduces its performance, which was already compromised by the elimination of amplifiers to make it that small. Larger ELRS recievers regain these amplifiers offering better reception and telemetry range at the cost of size and weight. The common size classes are shown below, but there are other recievers with slightly different sizes:
+
+    <figure>
+      <img class="center-img" src="../../assets/images/hw-selection/sizes.png"/>
+    </figure>
+
+    PP Sized Recievers (absolute smallest, lowest range regardless of antennas):
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:121]: "Receivers"

File: docs/hardware/hardware-selection.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+      * Namimno Flash RX
+
+    Larger Recievers (most feature rich but also significantly larger)
+
+      * Matek R24-D
+      * Radiomaster RP3
+      * BetaFPV SuperD
+
+## Transmitter Selection
+This selection lists out some common features of transmitters and some examples of transmitters with those features
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:142]: "This selection" or "This section"?

File: docs/hardware/hardware-selection.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+
+Everyone has different needs for a transmitter but the main needs come down to size, range and integration.
+
+??? info "Built-In Transmitters"
+
+    Certain manufacturers have made handsets with built in ELRS modules, which offer tigher integration with ELRS. These can be update through the handset as well as generally over WiFi as is standard.
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:148]: "tighter"

These can be "updated" through the handset...

File: docs/hardware/hardware-selection.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+
+    Some handsets with built in ELRS include:
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:150]: "built-in"

File: docs/hardware/hardware-selection.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+      * RadioMaster Zorro ELRS
+      * BetaFPV Lite Radio 3 Pro ELRS
+      * Jumper T-Pro
+      * Jumper T-Lite v2
+
+    A notable exception to this list is the iFlight Commando, which, while including a TX built into the handset is simply wired as an external module, and put within the casing. It does support an external module and notably has both 868/915MHz and 2.4GHz options available
+
+??? info "Frequency"
+
+    ExpressLRS offers both 2.4GHz and 900MHz systems, with each only working with receivers and transmitters of the same frequency. 2.4GHz is the currently most popular frequency given it's legality, featureset and cost. 2.4GHz links offer the fastest packet rates allowing for a more locked in feel while piloting and still offer massive amounts of range. 900MHz is the original long range frequency and still can be great for high wifi pollution environments with slightly better penetration.
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:162]: "feature set" or just "features"

"locked-in"

File: docs/hardware/hardware-selection.md [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+
+    For new users unless you plan on flying hundreds of kilometers or in a high noise environment we reccomend 2.4GHz hardware:
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:164]: Recommend is misspelled here

this section sounds like these are the ONLY ones we  recommend. Use "some of our recommendations" instead or any wording like that. ;)

File: docs/hardware/hardware-selection.md [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+    Micro modules will fit into the JR bay of a standard transmitter such as a TX16s or QX7, with some examples including:
+
+      * RadioMaster Ranger Micro
+      * Namimno Flash
+      * HappyModel ES24TX
+    
+    Nano modules will fit into a lite module bay such as that on a Zorro, T-Pro or X-Lite with some examples including:
+
+      * HappyModel ES24TX Slim Pro
+      * Radimoaster Ranger Nano
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:190]: RadioMaster is misspelled here

File: docs/hardware/hardware-selection.md [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+
+    Some notable exceptions that can fit into multiple or very specific module bays include:
+
+      * HappyModel ES24TX Lite - Fits onto Jumper T-Lite
+      * Axisflying Thor - Has a wire system to be able to plug into any radio with a CRSF output
+      * Radiomaster Ranger - includes Micro & Nano mounts and wiring for any CRSF compatible radio
+
+??? info "Power"
+
+    Most ELRS Transmitters have a 250mW or 500mW power cap, but for that extra percentage of range, a select few 1W modules exist, packing that punch needed to go even further. These transmitters include large heatsinks, fans and sometimes temperature sensing to cool the power output. Some of these 1W transmitters are:
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:202]: This sounds like there's massive range difference between a 500mW and 1W tx modules. Doubling power doesn't always mean double the range/penetration.
consider rewording to "needed to go a bit further".

"temperature sensing to cool the rf components"

File: docs/hardware/hardware-selection.md [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+
+      * HappyModel ES24TX Pro
+      * RadioMaster Ranger
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:205]: Ranger Micro and Ranger Nano are also 1W modules

File: docs/hardware/hardware-selection.md [hunk 11]
```diff
@@ -0,0 +1,278 @@
...
+      * Axisflying Thor
+      * BetaFPV Micro TX 1W
+      * RadioMaster Boxer
+      * Jumper T-Pro Internal
+      * NamimnoRC Flash OLED
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:210]: the older Namimno Flash non-OLED were 1W capable as well

File: docs/hardware/hardware-selection.md [hunk 12]
```diff
@@ -0,0 +1,278 @@
...
+      <img class="center-img" src="../../assets/images/hw-selection/screens.png"/>
+    </figure>
+
+    The only transmitter with a TFT screen on the market at the moment is the Axisflying Thor TX.
+
+    Some transmitters with an OLED include:
+
+      * Namimno Flash OLED
+      * RadioMaster Ranger
+      * BetaFPV Micro TX
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:226]: might want to include the Jumper Aion Nano module here.. Don't want anyone to feel left out.. :D

File: docs/hardware/hardware-selection.md [hunk 13]
```diff
@@ -0,0 +1,278 @@
...
+
+??? info "Backpack"
+
+    Most current transmitters with include a [backpack](../hardware/backpack/esp-backpack.md), which enables communication with accessories such as a VRX allowing for tighter integration with the transmitter and drone. Transmitters with a built in backpack include:
```
Comment (deadbytefpv) [inline docs/hardware/hardware-selection.md:230]: "will include"

"with a built-in... " (use a dash between built and in)

**Review by deadbytefpv — CHANGES_REQUESTED** [review]
Thank you for your work on this page.

Apologies for the late review. I've made several comments on the new page. Please have a look and consider those changes.

Might want to also add redirects for those two pages that were removed and consolidated into this one page. I didn't see an edit for the redirect.yml page

File: docs/hardware/hardware-selection.md [hunk 14]
```diff
@@ -0,0 +1,278 @@
+---
+template: main.html
+description: This will guide you on how to select the best ExpressLRS (ELRS) radio for your needs
+---
+
+![HW Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-hardware/master/img/hardware.png)
+
+ExpressLRS is hashtag blessed with the benefit of there being many receivers available from a variety of manufacturers. This begs the question "Which receiver is best?" There is no best receiver, there is only one that has the features you want for the price you want to pay in the size you want it. ExpressLRS does not recommend a specific brand or model, but provide the information to help you select the right receiver for your needs. The following page will list manufacturers and features to look out for on their receivers.
```
[the remark this answers is not in the document]
Comment (maybenikhil) [reply docs/hardware/hardware-selection.md:8]: generalized with hardware or just removed the words

File: docs/hardware/hardware-selection.md [hunk 15]
```diff
@@ -0,0 +1,278 @@
...
+  * **Racing Quads:** Size is again most important. Ceramic antennas could be less easily damaged, and the reduced range of tucking them inside the frame is fine due to the short flight range. An external 2.4GHz antenna dipole is still pretty easy to fit and can be tucked away for a small improvement over the ceramic, but comes with Chance of Choppage.
+  * **Freestyle Quads:** Minimum size is no longer an issue so Nano-sized receivers are the best bet here. A LNA is going to give you better reception behind obstacles. External antennas are a benefit as well, but you need to trade off how unobstructed the antenna will be versus getting it chopped. Diversity is of marginal benefit.
+  * **Long Range:** For sure you need an LNA, an external antenna, and a PA to extend the telemetry range. This isn't to say these are required for long range, 5km is achievable on a ceramic antenna receiver with no LNA/PA at 250Hz/100mW with a clear line of sight. Diversity is worth a lot more here. For planes without a flight controller, PWM recievers will work great. For absolute maximum range 900MHz can do more but 2.4GHz can still do 50+km.
```
[the remark this answers is not in the document]
Comment (maybenikhil) [reply docs/hardware/hardware-selection.md:37]: fair, I think the best case for diversity is actually short/midrange freestyle but yk... propoganda (will correct)

File: docs/hardware/hardware-selection.md [hunk 16]
```diff
@@ -0,0 +1,278 @@
...
+
+??? info "Frequency"
+
+    ExpressLRS offers both 2.4GHz and 900MHz systems, with each only working with receivers and transmitters of the same frequency. 2.4GHz is the currently most popular frequency given it's legality, featureset and cost. 2.4GHz links offer the fastest packet rates allowing for a more locked in feel while piloting and still offer massive amounts of range. 900MHz is the original long range frequency and still can be great for high wifi pollution environments with slightly better penetration.
```
[the remark this answers is not in the document]
Comment (maybenikhil) [reply docs/hardware/hardware-selection.md:41]: keeping locked in as two words

**Review by deadbytefpv — APPROVED** [review]

File: docs/hardware/hardware-selection.md (hunk 8 above)
Comment (deadbytefpv) [reply docs/hardware/hardware-selection.md]: You missed one.. :)

**Review by deadbytefpv — APPROVED** [review]
Sorry about this.. 

GH is saying there's still unresolved change request, but I looked and all's been resolved.

**Pull request merged**